components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -425,28 +623,68 @@ struct SignMessageRequest {
   string message;
 };
 
+// For default wallet setting, user assets, permission checks, importing
+// functionality and much more.
 interface BraveWalletService {
+  // Adds an observer for BraveWalletService
   AddObserver(pending_remote<BraveWalletServiceObserver> observer);
+
+  // Obtains the user assets for the specified chain ID
   GetUserAssets(string chain_id) => (array<ERCToken> tokens);
+
+  // Adds a user asset for the specified chain ID
   AddUserAsset(ERCToken token, string chain_id) => (bool success);
+
+  // Removes a user asset for the specified chain ID
   RemoveUserAsset(ERCToken token, string chain_id) => (bool success);
+
+  // Sets whether or not an asset is visible
   SetUserAssetVisible(ERCToken token, string chain_id, bool visible) => (bool success);
+
+  // Checks if the deprecated Crypto Wallets extension is installed
   IsCryptoWalletsInstalled() => (bool installed);
+
+  // Checks if MetaMask is installed (Used for knowing to present import UI)
   IsMetaMaskInstalled() => (bool installed);
+
+  // Imports from Crypto Wallets
   ImportFromCryptoWallets(string password, string newPassword) => (bool success,
-  string? errorMessage);
+      string? errorMessage);
+
+  // Imports from MetaMask
   ImportFromMetaMask(string password, string newPassword) => (bool success,
-  string? errorMessage);
+      string? errorMessage);
+
+  // Obtains the default wallet setting
   GetDefaultWallet() => (DefaultWallet default_wallet);
+
+  // Modifies the default wallet setting
   SetDefaultWallet(DefaultWallet default_wallet);
+
+  // Obtains the default base currency
   GetDefaultBaseCurrency() => (string currency);
+
+  // Sets the default base currency
   SetDefaultBaseCurrency(string currency);
+
+  // Obtains the default base cryptocurrency
   GetDefaultBaseCryptocurrency() => (string cryptocurrency);
+
+  // Sets the default base cryptocurrency
   SetDefaultBaseCryptocurrency(string cryptocurrency);
+
+  // Checks if the Ethereum permission is allowed for the origin on the account
   HasEthereumPermission(string origin, string account) => (bool success, bool has_permission);
+
+  // Resets the Ethereum permission for the account
   ResetEthereumPermission(string origin, string account) => (bool success);
+
+  // Obtains the active origin
   GetActiveOrigin() => (string origin);
+
+  // Obtains the pending sign message requests
   GetPendingSignMessageRequests() => (array<SignMessageRequest> requests);
+
   // UI to native
```

Comment:
We can enhance the comment like
"These are used for UI notifying native when the user approve/reject SignMessage[Hardware]Request"

Reply:
Updated